dbschema/project.esdl

```diff
@@ -28,19 +28,24 @@ module default {
           .step >= Project::Step.PendingFinanceConfirmation
         ) then ((
           with
-            fa := assert_exists(
-              __subject__.primaryLocation.fundingAccount,
-              message := "Project must have a primary location"
-            ),
-            existing := (
-              select detached Project.departmentId filter Project.primaryLocation.fundingAccount = fa
-            ),
-            available := (
-              <str>range_unpack(range(fa.accountNumber * 10000 + 11, fa.accountNumber * 10000 + 9999))
-              except existing
+            info := (
+              if __subject__ is MultiplicationTranslationProject
+                then (prefix := 8, startingOffset := 201)
```

Comment:
Was 80201 the right starting number?

Reply:
Just for github visibility: a comment on the story specifies 80201 would be the first: 
> Restrict 80200 and below (?)